Misc/NEWS.d/next/Library/2024-05-16-00-08-53.gh-issue-119067.1v2ny6.rst

```diff
@@ -0,0 +1,2 @@
+Add :const:`visible` to :mod:`string`, containing all non-blank ASCII characters
+(i.e., everything in :const:`printable` except for :const:`whitespace`).

```

Comment:
```suggestion
Add :const:`~string.visible` to :mod:`string`, containing all non-blank ASCII characters
(i.e., everything in :const:`~string.printable` except for :const:`~string.whitespace`).
```

Reply:
Apologies, I just pushed a commit without having seen your recent suggestions. I'll review them now.

Reply:
No problem. It's my fault for having posted a too early review (should have posted everything altogether).

Reply:
I've made those three corrections: fixing the issue reference, and the two instances of `:const:~string.visible`. Should I push that or hold off for anything else that might come up?

Many thanks for your patience, by the way.

Reply:
For now, I don't see anything. I think, it's still good to have a clickable link for `visible` in the WhatsNew entry, but I'd be intersted in Hugo's answer to know whether the Misc/NEWS entries can make use of the full Sphinx power or if they should be kept simple. I think you can simply commit whatever you have now, and we'll see when he replies.

> Many thanks for your patience, by the way.

It's more me that should be thankful since you kindly accept my back-and-forth way of reviewing.

Reply:
> It appears that the linter is not complaining here but I don't know whether we want to have a Sphinx-compliant output in MISC entries or not. @hugovk the guidelines for blurb tells me about an "rst" content but I see Sphinx content as well so I'm not sure whether it should only be pure ReST (docutils) or Sphinx.

They're Sphinx files.

The `Misc/` entries are eventually used by Sphinx. Another tool merges them all to a single file, and then Sphinx includes them into the changelog page:

* https://docs.python.org/3.14/whatsnew/changelog.html
* https://github.com/python/cpython/blob/main/Doc/whatsnew/changelog.rst

Reply:
I understand why there was only a single warning and not one due to the invalid NEWS entry:

https://github.com/python/cpython/blob/34f5ae69fe9ab0f5b23311d5c396d0cbb5902913/Doc/tools/check-warnings.py#L19-L23

Thank you Hugo!